src/main/java/com/redhat/engineering/srcclr/processor/CVEProcessor.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (C) 2018 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.redhat.engineering.srcclr.processor;
+
+import com.redhat.engineering.srcclr.SrcClrWrapper;
+import com.redhat.engineering.srcclr.json.Library;
+import com.redhat.engineering.srcclr.json.Record;
+import com.redhat.engineering.srcclr.json.SourceClearJSON;
+import com.redhat.engineering.srcclr.json.Vulnerability;
+import com.redhat.engineering.srcclr.utils.InternalException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.apache.commons.lang.StringUtils.isNotEmpty;
+
+public class CVEProcessor
+                implements ScanResult
+{
+    private final Logger logger = LoggerFactory.getLogger( CVEProcessor.class );
+
+    @Override
+    public ArrayList<Vulnerability> process( SrcClrWrapper parent, SourceClearJSON json ) throws InternalException
+    {
+        Record record = json.getRecords().get( 0 );
+        List<Library> libs = record.getLibraries();
+        ArrayList<Vulnerability> matched = new ArrayList<>( );
+
+        for ( Vulnerability vuln : record.getVulnerabilities() )
+        {
+            Library library = locateLibrary( libs, vuln );
+
+            if ( isNotEmpty ( vuln.getCve() ) )
+            {
+
+                // TODO: Call onto API with possible notification system
+
+                matched.add( vuln );
+                logger.info ( "Found vulnerability '{}' with identity {} in library {}:{}:{} and report is {}",
```

Comment:
I found this log really helpful when I analyzed the Thorntail results 👍  Now it logs out the ID(rather than CVSS), it's even better. Just a trivial stuff though, changing 'identify' to CVE ID or CVE identifier would be better, because that's the term used in the security field.